### DISCUSSION TITLE
Auto re-start a session after disconnect

### DISCUSSION THREAD

**Discussion by mastermetal08** [lathoub/Arduino-AppleMIDI-Library#112] [Q&A]
Hi,
When my session with iPad disconnects, I have to reset ESP32 (AP mode) and re-connect  wifi  and start over.
Is there a way to automatically restart session without having to reset ESP?

**Comment by lathoub** [comment]
Hi,

Thanks for signalling @mastermetal08 

Currently the underlying [MIDI library](https://github.com/FortySevenEffects/arduino_midi_library) has no end() or restart() method to restart the Transport layer.
Albeit the fix is the MIDI library is easy to make, it does have a significant impact on this library's dependancy on a specific version of the underlying MIDI library.

I [started a discussion](https://github.com/FortySevenEffects/arduino_midi_library/discussions/216) to investigate options
